requirements.txt

```diff
@@ -1,9 +1,12 @@
-pandas==2.0.2
-pymupdf~=1.24.3
-numpy~=1.24.3
-requests==2.32.3
 fastapi~=0.111.0
+llama-cloud-services~=0.6.1
+numpy~=1.24.3
+pandas==2.0.2
+parse-document-model==0.2.0
 pydantic~=2.9.0
 pydantic_settings~=2.2.1
-uvicorn==0.22.0
-parse-document-model==0.2.0
\ No newline at end of file
+python-magic-bin==0.4.14
```

Comment:
The docker build pipeline seems suggesting that there is a problem in python-magic-bin

```
Could not find a version that satisfies the requirement python-magic-bin==0.4.14
```